### ISSUE TITLE
Add Mylar to audo

### ISSUE THREAD

**Issue by lediable** [lsellens/script.service.audo#19]
Hi,

Can you please add Mylar to audo? If it's possible, of course :)

**Comment by lsellens** [comment]
Can you even view comic books in openelec?

**Comment by lediable** [comment]
Yup, with Aeon Nux I can read comics and with a shared folder I can use my tablet too
